src/components/Link/Link.js

```diff
@@ -26,8 +26,9 @@ export default class Link extends React.Component {
         onMouseEnter={this.props.mouseoverLink}
         onMouseLeave={this.props.mouseleaveLink}>
         <div className="chi-link__content">
-          {help}
+          {this.props.position === 'left' ? help : ''}
```

Comment:
```suggestion
          {this.props.position === 'left' ? help : null}
```
Did you try using null here? An empty string might cause problems